src/main/features/response/routes/claimant-dqs.ts

```diff
@@ -0,0 +1,18 @@
+import * as express from 'express'
+import { Paths } from 'response/paths'
+import { Claim } from 'claims/models/claim'
+import { DocumentsClient } from 'documents/documentsClient'
+import { ErrorHandling } from 'shared/errorHandling'
+import { DownloadUtils } from 'utils/downloadUtils'
+
+const documentsClient: DocumentsClient = new DocumentsClient()
+
+/* tslint:disable:no-default-export */
+export default express.Router()
+  .get(Paths.claimantDQs.uri,
+    ErrorHandling.apply(async (req: express.Request, res: express.Response): Promise<void> => {
+      const claim: Claim = res.locals.claim
+
+      const pdf: Buffer = await documentsClient.getClaimantHearingRequirementPDF(claim.externalId, res.locals.user.bearerToken)
+      DownloadUtils.downloadPDF(res, pdf, `${claim.claimNumber}-claim-form-defendant-copy`)
```

Comment:
this is downloading the claim form. Shouldn't this be the claimant response DQs document?

Reply:
it is downloading the right document, it was named claim-form-claimant-copy so I just changed it to defendant but I can change the name to something more suitable?